glmnet/tests/test_linear.py

```diff
@@ -63,6 +63,33 @@ def test_with_single_var(self):
         m = m.fit(x, y)
         self.check_r2_score(y, m.predict(x), 0.90)
 
+    def test_relative_penalties(self):
+        m1 = ElasticNet(random_state=4328)
+        m2 = ElasticNet(random_state=4328)
+        for h in self.inputs:
+            x, y = h
+            p = x.shape[1]
+
+            # m1 no relative penalties applied
+            m1.fit(x, y)
+
+            # find the nonzero indices from LASSO
+            nonzero = [ind for ind, z in enumerate(m1.coef_) if z != 0]
```

Comment:
[`np.nonzero`](http://docs.scipy.org/doc/numpy/reference/generated/numpy.nonzero.html) is the mode idiomatic way to do this